challenge.rb

```diff
@@ -6,7 +6,15 @@
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+  minimum_age = ()
+  age = ()
+  print "What is your age?"
+  age = gets.chomp
```

Comment:
No `gets` or `puts` needed here - `age` and `minimum_age` are being passed in as arguments.